src/debug/di/cordb.cpp

```diff
@@ -499,7 +506,7 @@ DbiGetThreadContext(HANDLE hThread,
     DT_CONTEXT *lpContext)
 {
     // if we aren't local debugging this isn't going to work
-#if !defined(_ARM_) || defined(FEATURE_DBGIPC_TRANSPORT_DI) || defined(__ANDROID__)
+#if !defined(_ARM_) || defined(FEATURE_DBGIPC_TRANSPORT_DI) || !HAVE_ALIGNED_MALLOC
```

Comment:
@parjong I guess I might have confused you. The reason why I was asking about it is that if we thought that we could have a scenario where the debugging would use a different transport, then I would rather fix the code below to not to use `_aligned_malloc` - the alignment requirement is small, so we can use regular malloc, allocate buffer larger by 15 bytes and place the context to an aligned address in this buffer. 
But if we think we will not need it, then ifdefining it out for all non-windows platforms is better. 
My thinking stemmed from your change that defined the HAVE_ALIGNED_MALLOC and my feeling was that the missing aligned malloc is not the actual reason why we ifdef out the code, since that could be easily fixed. The real reason is that there is no point in having the code compiled on Unix or Android. 
So based on these thoughts, I would prefer adding `|| defined(PLATFORM_UNIX)` to the condition instead of introducing the HAVE_ALIGNED_MALLOC.